xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -102,6 +111,14 @@ bool CPVRChannelGroups::Update(const CPVRChannelGroup &group, bool bUpdateFromCl
   return true;
 }
 
+void CPVRChannelGroups::SortGroups()
+{
+  CSingleLock lock(m_critSection);
+
+  // sort groups by position
+  std::sort(m_groups.begin(), m_groups.end(), compGroupPosition);
```

Comment:
Since we can use C++11 now there's no need to pollute the namespace with a comparator function.

```
std::sort(m_groups.begin(), m_groups.end(), 
    [](const CPVRChannelGroupPtr &left, const CPVRChannelGroupPtr &right)
{
    return left->GetPosition() < right->GetPosition();
}
```